qt/python/mantidqt/mantidqt/widgets/sliceviewer/presenters/base_presenter.py

```diff
@@ -121,3 +121,7 @@ def region_selection(self, state):
     @abc.abstractmethod
     def get_extra_image_info_columns(self, xdata, ydata):
         pass
+
+    @abc.abstractmethod
+    def is_integer_frame(self):
```

Comment:
From the nice description on this PR I get why it's necessary to define many abstract methods in this class, but in this particular instance given `is_integer_frame` only applies to one specific type of MD workspace would it make sense just to make this a normal method that's overridden in the sliceviewer presenter rather than having to re-define it in other presenters?